mysql/src/main/scala/zio/sql/mysql/MysqlSqlModule.scala

```diff
@@ -36,8 +36,8 @@ trait MysqlSqlModule extends Sql { self =>
     val Log10       = FunctionDef[Double, Double](FunctionName("log10"))
     val MakeTime    = FunctionDef[(Int, Int, Double), LocalTime](FunctionName("maketime"))
     val Now         = FunctionDef[Any, ZonedDateTime](FunctionName("now"))
-    val Pi          = Expr.FunctionCall0[Double](FunctionDef[Any, Double](FunctionName("pi")))
     val RPad        = FunctionDef[(String, Int, String), String](FunctionName("rpad"))
+    val Pi          = Expr.FunctionCall0[Double](FunctionDef[Any, Double](FunctionName("pi")))
```

Comment:
this switch of `Pi` was not necessary :)

Reply:
ah ok I see now its aranged in alphabetical order. I was trying group all FunctionCall together. will undo

Reply:
fixed now